README.md

```diff
@@ -1,5 +1,15 @@
-`jmpost` is currently under development and not yet in a testable state.
 
-The scope of the package is to enable "Joint Models Predicting Overall Survival Trajectories" based on non-linear mixed effects models for the tumor size over time and correlating that with overall survival. We implement this within the package in stan and provide a flexible user interface.
+`jmpost` is currently under development and not yet testable.
 
+The goal of the `jmpost` package is to fit joint models involving:
+1. a parametric time-to-event sub-model,
+2. a nonlinear (or linear) mixed-effect sub-model, describing individual time profiles (_i.e._ trajectories) for a continuous marker,
+3. a link function (_a.k.a._ association term).
+More specifically, the model implemented in this package utilizes a modeling framework described previously **[1-3]** to link overall survival to tumor size data in oncology clinical trials, sponsored by Roche-Genentech.
```

Comment:
do we need Roche-Genentech mentioned here? The models would also be applicable to other sponsors?